docs/manual/src/tutorial/callback_interfaces.md

```diff
@@ -11,34 +11,35 @@ in Rust.
 
 # Using callback interfaces
 
-1. Define a Rust trait. 
+## 1. Define a Rust trait.
```

Comment:
![Screen Shot 2021-10-04 at 4 09 13 PM](https://user-images.githubusercontent.com/311152/135936478-c67fd5e8-f9dd-4475-84f9-b1ebb2cc7d6f.png)
FYI - I'm seeing this when previewing the file on Github.